docs/building-apps/moneygram-access-integration-guide.mdx

```diff
@@ -185,10 +216,27 @@ You will need the following pieces of information:
 - The amount the user would like to withdraw / cash-out
   - This should be collected from the user prior to initiating this transaction
 
-The following code can be used as a reference for implementing this logic yourself. This code is not necessarily production-ready.
+The following code can be used as a reference for implementing this logic yourself.
 
 <CodeExample>
 
+```ts
+// Use same "anchor" object from previous step.
+const { url, id } = await anchor.sep24().withdraw({
+  authToken: authToken, // Use same "authToken" string from previous step
+  withdrawalAccount: FUNDS_STELLAR_KEYPAIR.public_key,
+  assetCode: ASSET_CODE, // Should be "USDC"
```

Comment:
I think we should follow our own guidelines from [SEP-24 guide](https://developers.stellar.org/docs/building-apps/wallet/sep24) 
```
const asset = info.currencies.find(({ code }) => code === "USDC").assetId;
```

Reply:
@Ifropc this code appears to be broken. We can't access `assetId` directly from `info.currencies`.

I think this is the best we could do with current TS Wallet SDK:
```
    const tomlInfo = await anchor.getInfo();

    const tomlAsset = tomlInfo.currencies.find(({ code }) => code === "USDC");

    if(!tomlAsset?.code || !tomlAsset?.issuer) {
      throw new Error("Anchor does not support USDC asset or is not correctly configured on TOML file");
    }

    const asset = new IssuedAssetId(
      tomlAsset.code,
      tomlAsset.issuer,
    );
```

What do you think?

Reply:
@Ifropc I'm wondering if we should display this same code above 👆  in our [SEP-24 guide](https://developers.stellar.org/docs/building-apps/wallet/sep24) in place of the broken `const asset = info.currencies.find(({ code }) => code === "USDC").assetId;` code. Wdyt?

Reply:
https://github.com/stellar/stellar-docs/pull/658/commits/295515b170ec57706bff7d542ac9e2a0398b0334